config/config.php

```diff
@@ -34,7 +34,6 @@
             __DIR__ . '/../src/ValueObject',
             __DIR__ . '/../src/TypoScript/Conditions',
             __DIR__ . '/../src/TypoScript/Visitors',
-            __DIR__ . '/../src/Yaml/Form/Transformer',
```

Comment:
This directory does not exist anymore

Reply:
Yes, should be Rector instead of Transformer. PhpStorm is not refactoring this. Shit. Also i found a naming issue with EmailFinisherRectorInterface

Reply:
Should be EmailFinisherRector without the Suffix. PhpStorm was not accurate here.

Reply:
Yea, luckily we have PHPStan and Rector to cover us :)

Reply:
Fixed it in a follow up commit.

Reply:
I'll add the correct ignore and suffix in next PR. This is already big one

Reply:
I have done that already.

Reply:
Ok, I'll rebase :)